src/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -5,20 +5,10 @@ namespace System.Runtime.InteropServices.RuntimeInformationTests
 {
     public class CheckPlatformTests
     {
-        [Fact, PlatformSpecific(PlatformID.Windows)]
-        public void CheckWindows()
-        {
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Create("WINDOWS")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("windows")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("Windows NT")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Linux));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
-        }
-
         [Fact, PlatformSpecific(PlatformID.Linux)]
         public void CheckLinux()
         {
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
```

Comment:
Done 5db6e7e.
I will send a PR for FreeBSD to https://github.com/dotnet/buildtools and add back the test. Is it alright?